packages/sw-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -178,6 +178,18 @@ class RequestWrapper {
   async fetch({request}) {
     assert.atLeastOne({request});
 
+    if (this.behaviorCallbacks.requestWillFetch) {
+      for (let callback of this.behaviorCallbacks.requestWillFetch) {
```

Comment:
`this.behaviorCallbacks` was renamed to `this.pluginCallbacks` (see https://github.com/GoogleChrome/sw-helpers/pull/203).

Sorry!